database/migrations/2022_05_17_155546_create_life_events_table.php

```diff
@@ -74,7 +74,8 @@ public function up()
             $table->unsignedBigInteger('currency_id')->nullable();
             $table->unsignedBigInteger('paid_by_contact_id')->nullable();
             $table->integer('duration_in_minutes')->nullable();
-            $table->integer('distance_in_km')->nullable();
+            $table->integer('distance')->nullable();
+            $table->string('distance_unit')->nullable();
```

Comment:
This unit can be a 2 characters long.
```suggestion
            $table->char('distance_unit', 2)->nullable();
```